latex/delegation.tex

```diff
@@ -333,6 +431,39 @@
 
 \end{figure}
 
+
+The rules for updating the part of the ledger state defining the current stake
+pools are given in \cref{fig:pool-rules}. Note
+that in all three rules, the current epoch is part of the context. It is not
+necessary to include $cepoch$ in the context for Pool-Reg. However,
+we decided to include it for consistency and composability of stake pool rules.
+
+For each rule, again, we first check that a given certificate $c$ is of
+the correct type using the
+corresponding $\RegPool$ and $\RetirePool$ predicates.
+
+The first rule, Pool-Reg, can be used to register a new stake pool, register a
+stake pool under a different author ($\fun{author} c$), or stop the retirement
+process for that key (the hash key and associated planned retirement epoch
+ will be removed from the $retiring$ parameter).
+
+The second rule, Pool-Retire, starts the pool retirement process. Given an
+epoch $e$, the application of this rule requires that this epoch is in
+the future (after $cepoch$), as well as that it is
+and less than $\emax$ epochs after the current one, and also that
+the certificate author's key is in the list of registered pools.
+This rule simply adds the pair of the key of the
+certificate author and the planned retirement epoch $e$ to the $retiring$
+parameter.
+
+The third rule, Pool-Reap, is used when the planned retirement epoch $e$ for
+a given stake pool in the $\fun{retiring}$ set is reached. In this case, the
+pair associated with the $stpools$
+
```

Comment:
Somehing seems to be missing here

Reply:
Fixed the last sentence:
In this case, all the
pairs associated with hash keys of pools which are scheduled to retire in this
epoch are removed
from both $\var{stpools}$ and $\var{retiring}$.

And added a note:
\begin{note}
$\var{retired}$ and $\var{retiring}$ are both finite maps. For domain exclusion,
we only need the hashes of $\var{retired}$?
\end{note}